app/controllers/users_controller.rb

```diff
@@ -1,11 +1,6 @@
 class UsersController < ApplicationController
   def index
     @users = User.all
-
-    respond_to do |format|
-      format.html # index.html.erb
-      format.json { render json: @users }
-    end
   end
 
   def show
```

Comment:
- [ ] [OPTIONAL]
An error is thrown in an attempt to sign out. Let's endeavor to improve upon this aspect.

**Signout Error:**
![image](https://github.com/juaryimami/blog-app/assets/12745474/0e77c01a-f817-4995-9769-64b94584503e)